calculate_risk_percentiles function raises ZeroDivisionError

```python
    def test_calculate_risk_percentiles_for_TPR():
        risks = np.array([1.0, 0.89, 0.44, 0, 0.78, 0.67, 0.56, 0.33, 0.22, 0.11])
        points = np.array([0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0])
        calculate_risk_percentiles = np.frompyfunc(
            lambda p: np.count_nonzero(risks <= p) / len(risks), 1, 1)
        actual = calculate_risk_percentiles(points)
        expect = np.array([0.0, 0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 1.0])
>       np.testing.assert_almost_equal(actual, expect, decimal=4)
E       ZeroDivisionError: float division by zero
```

--- a/predictiveness_curve/base.py
+++ b/predictiveness_curve/base.py
@@ -124,19 +124,29 @@ def plot_predictiveness_curve(risks,
     num_positive = labels.sum()
 
     if kind.upper() == 'TPR':
-        calculate_risk_percentiles = np.frompyfunc(
-            lambda p: np.count_nonzero(risks <= p) / len(risks), 1, 1)
+
+        def f(point):
+            count = np.count_nonzero(risks <= point)
+            return count / len(risks) if count > 0 else 0
+
+        calculate_risk_percentiles = np.frompyfunc(f, 1, 1)
         risk_percentiles = calculate_risk_percentiles(points)
         risk_percentiles = np.append(0, risk_percentiles)
         points = np.append(0, points)
+
     elif kind.upper() == 'EF':
+
+        def f(point):
+            count = np.count_nonzero(risks >= point)
+            return count / len(risks) if count > 0 else 0
+
         labels = labels[::-1]
         risks = risks[::-1]
-        calculate_risk_percentiles = np.frompyfunc(
-            lambda p: np.count_nonzero(risks >= p) / len(risks), 1, 1)
+        calculate_risk_percentiles = np.frompyfunc(f, 1, 1)
         risk_percentiles = calculate_risk_percentiles(points)
         risk_percentiles = np.append(risk_percentiles, 0)
         points = np.append(points, 1)
+
     else:
         raise ValueError(f'kind must be either TPR or EF, not {kind}')
 

--- a/tests/test_predictiveness_curve.py
+++ b/tests/test_predictiveness_curve.py
@@ -172,6 +172,133 @@ def test_predictiveness_curve():
     assert actual == expect
 
 
+def test_display_arguments():
+    scores = np.array([10, 9, 5, 1, 8, 7, 6, 4, 3, 2])
+    labels = np.zeros((10, ), dtype='int32')
+    labels[:4] = 1
+
+    fig = plot_predictiveness_curve(scores,
+                                    labels,
+                                    normalize=True,
+                                    fontsize=20,
+                                    xlabel='percentiles',
+                                    top_ylabel='Risk value',
+                                    bottom_ylabel='TPR value',
+                                    kind='TPR')
+    axes = fig.get_axes()
+
+    actual = len(axes)
+    expect = 2
+    assert actual == expect
+
+    ax = axes[0]
+    actual = ax.get_xlim()
+    expect = (-0.03, 1.03)
+    assert np.array_equal(actual, expect)
+
+    actual = ax.get_ylim()
+    expect = (-0.03, 1.03)
+    assert np.array_equal(actual, expect)
+
+    actual = ax.xaxis.label.get_fontsize()
+    expect = 20.0
+    assert actual == expect
+
+    actual = ax.yaxis.label.get_fontsize()
+    expect = 20.0
+    assert actual == expect
+
+    actual = ax.xaxis.get_label_text()
+    expect = ''
+    assert actual == expect
+
+    actual = ax.yaxis.get_label_text()
+    expect = 'Risk value'
+    assert actual == expect
+
+    ax = axes[1]
+    actual = ax.get_xlim()
+    expect = (-0.03, 1.03)
+    assert np.array_equal(actual, expect)
+
+    actual = ax.get_ylim()
+    expect = (-0.03, 1.03)
+    assert np.array_equal(actual, expect)
+
+    actual = ax.xaxis.label.get_fontsize()
+    expect = 20.0
+    assert actual == expect
+
+    actual = ax.yaxis.label.get_fontsize()
+    expect = 20.0
+    assert actual == expect
+
+    actual = ax.xaxis.get_label_text()
+    expect = 'percentiles'
+    assert actual == expect
+
+    actual = ax.yaxis.get_label_text()
+    expect = 'TPR value'
+    assert actual == expect
+
+    fig = plot_predictiveness_curve(scores, labels, kind='EF')
+    axes = fig.get_axes()
+
+    actual = len(axes)
+    expect = 2
+    assert actual == expect
+
+    ax = axes[0]
+    actual = ax.get_xlim()
+    expect = (-0.03, 1.03)
+    assert np.array_equal(actual, expect)
+
+    actual = ax.get_ylim()
+    expect = (-0.03, 1.03)
+    assert np.array_equal(actual, expect)
+
+    actual = ax.xaxis.label.get_fontsize()
+    expect = 14.0
+    assert actual == expect
+
+    actual = ax.yaxis.label.get_fontsize()
+    expect = 14.0
+    assert actual == expect
+
+    actual = ax.xaxis.get_label_text()
+    expect = ''
+    assert actual == expect
+
+    actual = ax.yaxis.get_label_text()
+    expect = 'Risk'
+    assert actual == expect
+
+    ax = axes[1]
+    actual = ax.get_xlim()
+    expect = (-0.03, 1.03)
+    assert np.array_equal(actual, expect)
+
+    actual = np.array(ax.get_ylim())
+    expect = np.array([0.7500, 2.5833])
+    np.testing.assert_almost_equal(actual, expect, decimal=4)
+
+    actual = ax.xaxis.label.get_fontsize()
+    expect = 14.0
+    assert actual == expect
+
+    actual = ax.yaxis.label.get_fontsize()
+    expect = 14.0
+    assert actual == expect
+
+    actual = ax.xaxis.get_label_text()
+    expect = 'Risk percentiles'
+    assert actual == expect
+
+    actual = ax.yaxis.get_label_text()
+    expect = 'EF'
+    assert actual == expect
+
+
 def test_kind_error():
     with pytest.raises(ValueError,
                        match='kind must be either TPR or EF, not XXX'):
@@ -179,6 +306,27 @@ def test_kind_error():
         labels = np.zeros((10, ), dtype='int32')
         labels[:4] = 1
         _ = plot_predictiveness_curve(scores, labels, kind='XXX')
+
+
+def test_calculate_risk_percentiles_for_TPR():
+    def f(point):
+        count = np.count_nonzero(risks <= point)
+        return count / len(risks) if count > 0 else 0
+
+    calculate_risk_percentiles = np.frompyfunc(f, 1, 1)
+
+    risks = np.array([1.0, 0.89, 0.44, 0, 0.78, 0.67, 0.56, 0.33, 0.22, 0.11])
+    points = np.array([0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0])
+    actual = calculate_risk_percentiles(points)
+    expect = np.array([0.1, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0])
+    np.testing.assert_almost_equal(actual, expect, decimal=4)
+
+    risks = np.array(
+        [1.0, 0.89, 0.44, 0.10, 0.78, 0.67, 0.56, 0.33, 0.22, 0.11])
+    points = np.array([0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0])
+    actual = calculate_risk_percentiles(points)
+    expect = np.array([0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0])
+    np.testing.assert_almost_equal(actual, expect, decimal=4)
 
 
 if __name__ == '__main__':

